bundles/org.openhab.binding.knx/README.md

```diff
@@ -235,18 +235,23 @@ For UoM support see the explanations of the `number` channel.
 #### Group Address Notation
 
 ```text
-<config>="[<dpt>:][<]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
+<config>="[<dpt>:][<>]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
```

Comment:
Wouldn't the proper notation be
```
<config>="[<dpt>:][<|>]<mainGA>[[+[<]<listeningGA>][+[<]<listeningGA>..]]"
```

Reply:
Yes, only one of both modifiers is „allowed“, but not sure if that is needed as only one makes sense.

In case we want to note that, I would prefer to have a short sentence saying that, because | is not-exclusive OR, but in fact it is < XOR >.

Reply:
I know, it is not a formal syntax, but the ' |' is supposed to say it is one of the chars that goes there (if any). At the moment it looks as if it is optional to put `<>` in there (and nothing else).

Reply:
An alternative would be to say `[<][>]` as is said that the parts in brackets are optional. For that we would definitely need to say it is either … or ….

WDYT is easier to read and more “intuitive” for the end user?

Reply:
' [<][>]` would clearly indicate that both are optional, but independent. That's why I suggested `[<|>]`, which is imho a quite common way to show the alternatives that can go into the optional placeholder.

Reply:
Using `|` as "one or the other" is also the Java regex style, so I would vote for that.

Reply:
Maybe we should change the style to mark variables '''<dpt>''',  as the arrows have a different meaning.
dpt should do, and I think using | for or is intuitive for most users....

Reply:
No, I would leave the variables as `<variable>` - imho this is very common and easy to understand.